ds_judgements_public_ui/sass/includes/_recent_judgments.scss

```diff
@@ -5,13 +5,30 @@
     padding-bottom: $space-8;
   }
 
+  &__container-beta {
+    padding-top: $space-3;
+    padding-bottom: $space-8;
+    @media (max-width: $grid-breakpoint-medium) {
+      padding-top: 0;
+    }
+
+    h2 {
+      margin-top: $space-2;
+    }
+  }
+
   &__header {
     @include heading_text;
     font-size: 1.8rem;
     padding-bottom: $space-6;
     border-bottom: 5px solid $color-yellow;
   }
 
+  &__header-beta {
+    @include heading_text;
+    font-size: 1.8rem;
```

Comment:
Is `$typography-xl-text-size` too big here?

Reply:
updated although the 2rem is slightly larger than 1.8rem its for testing so should be fine we will get feedback @jlhdxw